src/bot/handlers/commentHandler.js

```diff
@@ -0,0 +1,26 @@
+const debug = process.env.NODE_ENV === "development";
+
+module.exports = class CommentHandler {
+    constructor(comment, logger) {
+        this.comment = comment;
+        this.logger = logger;
+    }
+
+    async reply(text) {
+        if (debug) {
+            this.logger.info("Not replying because you are developing");
+            return false;
+        }
+        let result = false;
+        this.logger.info("Replying now");
+        this.comment
+            .reply(text)
+            .then(() => {
+                this.logger.info("Text of reply was: " + text);
+                result = true;
+            })
+            .catch(this.logger.error);
+        return result;
```

Comment:
Das klappt so nicht, habe es auch nochmal getestet. Zum Zeitpunkt, in dem diese Zeile ausgeführt wird, wurde der asynchrtone `then` Block oben noch nicht ausgeführt. Das bedeutet, dass result hier immer `false` wäre